SCons/Tool/JavaCommonTests.py

```diff
@@ -674,32 +672,51 @@ def test_jdk_globs(self):
         Verify that the java path globs work with specific examples.
         :return:
         """
-        from SCons.Tool.JavaCommon import java_linux_include_dirs_glob, java_linux_version_include_dirs_glob, java_win32_dir_glob, java_win32_version_dir_glob, java_macos_include_dir_glob, java_macos_version_include_dir_glob
+        from SCons.Tool.JavaCommon import (
+            java_linux_include_dirs_glob,
+            java_linux_version_include_dirs_glob,
+            java_win32_dir_glob,
+            java_win32_version_dir_glob,
+            java_macos_include_dir_glob,
+            java_macos_version_include_dir_glob,
+        )
 
         # Test windows globs
         win_java_dirs = [
-            ('C:/Program Files/Java/jdk1.8.0_201/bin', '1.8.0'),
-            ('C:/Program Files/Java/jdk-11.0.2/bin', '11.0.2'),
-            ('C:/Program Files/Java/jdk1.7.0_80/bin', '1.7.0')
+            (r'C:/Program Files/Java/jdk1.8.0_201/bin', '1.8.0'),
```

Comment:
you dropped 1.7.0?

Reply:
This is just made-up test patterns to match against the ones in the tool (doesn't even go out to the filesystem for this check), so I updated the list with ones taken off a system where I installed extra Javas so they're realistic. Didn't see the need to keep two patterns that were really similar like 1.7 and 1.8 were in the original.

Reply:
gotcha